src/clj/witan/send/parquet.clj

```diff
@@ -94,6 +97,10 @@
   saturate the cores of whatever machine is running the simulation.
   "
   [{:keys [config] :as projection-config}]
+  (when (false? (.exists (io/file "resources")))
+    (.mkdir (java.io.File. "resources")))
+  (io/copy (io/file (io/resource "logback.xml"))
+           (io/file "resources/logback.xml"))
```

Comment:
I don't think this will work as the logback.xml won't get loaded from resources. I'll have to check that though. We could have it throw and exception with a message to say that you should put in the file and re-run everything.

Reply:
ah I understand your concern now with your fix!